.github/workflows/version.yml

```
@@ -0,0 +1,69 @@
+
+      - name: Create Release Pull Request or Publish
+        uses: changesets/action@v1
+        with:
+          publish: ./scripts/release.sh
+          title: "Release: Extension Version Packages"
+          commit: "chore: release extension packages"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          VSCE_PAT: ${{ secrets.VSCE_PAT }}
+          OVSX_PAT: ${{ secrets.OVSX_PAT }} 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)